03-a_review_of_r_modeling_fundamentals.Rmd

```diff
@@ -98,11 +98,17 @@ inter_fit <- lm(Volume ~ Girth * Height, data = trees)
 inter_fit
 ```
 
-Same goes for polynomial terms.
+Same goes for polynomial terms. The use of the identity function, `I()`, allows us to apply literal math to the predictors.
 
 ```{r }
 poly_fit <- lm(Volume ~ Girth + I(Girth^2) + Height, data = trees)
-poly_fit
+poly_fit 
+```
+
+```{r }
+poly_fit3 <- lm(Volume ~ (.)^2, data = trees)
+poly_fit3
+#There are only two predictors in this model so this produces the same results as inter_fit but if there were 3 there would be three individual effects and the combination of those effects as interaction depending on if the third variable is continuous or categorical
```

Comment:
Slight style tweaks:

* Add a space between # and There
* Type ctrl-shift-/ (or cmd-shift-/) in RStudio to reflow the comment (that keeps it from going off the side of the page if the browser doesn't wrap it).